src/firebase/index.js

```diff
@@ -0,0 +1,35 @@
+import firebase from "firebase/compat/app";
```

Comment:
Not sure why we need a separate firebase folder. just good organisation I guess?

Having a separate firebase.js file is similarly just nice organisation and separation of concerns

This file's job is to contain all the config and import all the relevant modules from webpack